CalendarApp/BookingRoom/Controllers/DisplayController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using BookingRoom.Models;
+using BookingRoom.Models.Displaying;
+
+namespace BookingRoom.Controllers
+{
+    public class DisplayController : ApiController
+    {
+        private readonly CalendarConnection _connection;
+
+        public DisplayController()
+        {
+            _connection = new CalendarConnection();
+        }
+
+        [HttpGet]
+        public HttpResponseMessage DisplayEvents()
+        {
+            var calendars = _connection.GoogleCalendar.CalendarList.List().Execute().Items;
+            var eventsToDisplay = new List<EventDto>();
+
+            foreach (var calendarEntry in calendars)
```

Comment:
I'd suggest to replace list instantiation and adding items to it with a single linq query.
Again that double call to googleAPI seems wrong